packages/frontend/src/pages/mode-selection/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useIntl } from 'react-intl';
+import { Col, Row, Typography } from 'antd';
+
+import { FightMode } from './const';
+import { useModeSelection } from './hooks';
+import ModeCard from '../../components/ModeCard';
+import { SkywalkerImage, StarshipImage } from '../../assets/images';
+
+const ModeSelectionPage = () => {
+  const intl = useIntl();
+  const { onSelectMode } = useModeSelection();
+
+  return (
+    <>
+      <Typography.Title style={{ textAlign: 'center' }}>
+        {intl.formatMessage({
+          id: 'mode.select',
+          defaultMessage: 'Select mode',
+        })}
```

Comment:
in JSX you should use `<FormattedMessage />` rather than `intl.formatMessage...`